### PR TITLE
doc: Record the steps used to update server software

### PR DESCRIPTION
**Pull request by gwhitney**
When I went to install the latest version of the software on the production server, I realized there were not explicit steps for doing so recorded in the documentation. This PR begins to fill that gap, with the knowledge I have gleaned so far.

### CODE REVIEW

File: doc/server-administration.md [hunk 1]
```diff
@@ -60,6 +60,33 @@ Stop `numberscope`:
...
+1. Log into the server and `cd ~scope/repos/backscope`.
+
+2. `sudo -u scope git pull`
+
+3. `sudo systemctl restart numberscope`
+
+You can then try `sudo systemctl status numberscope` to see if the service
+thinks it is running correctly, and you can point your browser for example at
+`https://numberscope.colorado.edu/api/get_commit` (Warning: not working as of
+commit 4ebfc9c) or
```
Comment (katestange) [inline doc/server-administration.md:79]: does "not working as of commit N" mean it is currently not working, and was checked at commit N, or does it mean it only started working at commit N, or something else?

Comment (gwhitney) [reply doc/server-administration.md:79]: Currently not working, see #108

**Comment by gwhitney** [comment]
I just did (to generate this), but there's no harm in re-doing. Or you could merge this, and then do that, so that it won't be the null update. Whichever you prefer. Thanks!

**Review by Vectornaut — APPROVED** [review]
I've logged into the server and tried all of the added instructions. They all behave as I'd expect. The example request also works as expected.

**Pull request merged**